resip/stack/test/CMakeLists.txt

```diff
@@ -59,6 +59,7 @@ test(testTimer testTimer.cxx)
 if(NOT WIN32)
 manual_test(testTransactionFSM testTransactionFSM.cxx TestSupport.cxx)
 endif()
+test(testTransportSelector testTransportSelector.cxx)
```

Comment:
A question to the reviewer or maintainer. Shouldn't this test be run for the Windows platform as it has been done for testTransactionFSM?
I don't know these requirements as I've tested on Linux.

Reply:
I'm not sure the history, but testTransactionFSM does not build on Windows, so it is excluded from Windows builds - ideally this would be fixed.  The old testTransportSelector won't compile on Windows either, I suspect it won't compile on linux either, since it is trying to construct TransportSelector with the wrong number of arguments.  Once this change get's accepted, I'll see if the new testTransportSelector will compile/run on windows, and if not I can exclude it for windows until someone has cycles to address.

Reply:
See the restored tests here - #364